src/app/pages/login/login-form/login-form.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import { LoginService } from '../../../services/login/login.service';
+import { LoginData } from '../../../models/login/loginData';
+import { ToastrService } from 'ngx-toastr';
+
+@Component({
+  selector: 'app-login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.scss']
+})
+export class LoginFormComponent implements OnInit {
```

Comment:
@BrianSerem, I realize the login page has the generic LandVille title, I think we should customize the metadata of the page especially the title, to something like `LandVille Login' or something more descriptive. We could achieve this by passing this metadata in the Login Route configurations in our Routing Module.